code/modules/bounty/presets/food_drink.dm

```diff
@@ -31,3 +31,174 @@
+	individual_reward = 18
+
+	days_until_expiry = 1
+
+/datum/bounty/food/doorsprint_05
+	name = "DoorSprint Order: Pizza Mozzarella"
+	author = "Jonathan Zepelli"
+	description = "I made this new song and I need some unsliced pizzas to go along with it. Would you like to hear a sample? It goes like this... \
+	Pizza mozzarella... Rella rella... Pizza mozza... Mozza... Rella, rella, rella. Rella!"
+
+	items_wanted = list(/obj/item/weapon/reagent_containers/food/snacks/sliceable/pizza/margherita = 3)
+
+	department_reward = 60
+	individual_reward = 30
+
+	days_until_expiry = 1
+
+/datum/bounty/food/doorsprint_06
+	name = "DoorSprint Order: Why Don't You Have Pork Buns?"
+	author = "Matt Yu"
+	description = "My father always told me that a man who has never had a pork bun is not a whole man. I want to surprise him with a whole bunch of \
+	delicious meat buns."
+
+	items_wanted = list(/obj/item/weapon/reagent_containers/food/snacks/meatbun = 12)
+
+	department_reward = 120
+	individual_reward = 60
+
+	days_until_expiry = 1
+
+/datum/bounty/food/doorsprint_07
+	name = "DoorSprint Order: Minty Fresh?"
+	author = "Saul Fellow"
+	description = "The boys and I are heading down to the lake for some good times and a couple of drinks. We need a very large pitcher of Mint Julep. \
+	Honestly, you could just toss it all into a bucket and send it our way. Thanks in advance."
+
+	reagents_wanted = list("mintjulep" = 120)
+
+	department_reward = 30
+	individual_reward = 15
+
+	days_until_expiry = 1
+
+/datum/bounty/food/doorsprint_08
+	name = "DoorSprint Order: Stress Free"
+	author = "Amy Amelia Amisdottir"
+	description = "I've finally completed all of my exams! It's time to get wasted before graduation! I'm hosting a massive pre-grad party and I need something \
+	new and delicious. I was looking through the DoorSprint catalogue and saw something called a Clover Club. Let's try that!"
+
+	reagents_wanted = list("cloverclub" = 240)
+
+	department_reward = 60
+	individual_reward = 30
+
+	days_until_expiry = 1
+
+/datum/bounty/food/doorsprint_09
+	name = "DoorSprint Order: I'll Live Forever, Why Won't You Too?"
+	author = "Rue Cole"
+	description = "My dad really used to like drinking Euphorias before he uh... went away. I never understood why he liked these things so much but it's the only \
+	way I can still feel like he's here... The most expensive cocktail in the universe but it can't cure my nostalgia for the past."
+
+	reagents_wanted = list("euphoria" = 30)
+
+	department_reward = 65
+	individual_reward = 32
+
+	days_until_expiry = 1
+
+/datum/bounty/food/doorsprint_10
+	name = "DoorSprint Order: Whatcha Drinkin'?"
+	author = "Walton Oreole"
+	description = "Look at that subtle off-white colouring. The tasteful thickness of it. Oh my God, it even has salt on the rim. You can't tell through the text \
+	but I'm sweating. I need one. Now!"
+
+	reagents_wanted = list("margarita" = 30)
+
+	department_reward = 30
+	individual_reward = 15
```

Comment:
margarita pizzas are quite tedious to make. up the amount for individuals/dept on this one or reduce the amount of pizzas.

Reply:
This is the margarita drink, not the pizza

Reply:
Lmao forgive me then, I can be like this. I'll merge this now.